src/test/java/com/uber/cadence/activity/LocalActivityContextPropagationTest.java

```diff
@@ -0,0 +1,242 @@
+package com.uber.cadence.activity;
```

Comment:
The tests looks good. 
Could you also run `./gradlew licenseFormat` to add license header for this file, before we can merge it?
Thanks

Reply:
License updated
`
Updating license header in src\test\java\com\uber\cadence\activity\LocalActivityContextPropagationTest.java
1 license header(s) updated. A backup of the original file(s) was created in build\tmp\updateLicenseTest\original
`